Products/CMFPlone/browser/search.py

```diff
@@ -214,7 +214,7 @@ def breadcrumbs(self, item):
             return None
         if len(breadcrumbs) > 3:
             # if we have too long breadcrumbs, emit the middle elements
-            empty = {'absolute_url': '', 'Title': six.text_type('…', 'utf-8')}
+            empty = {'absolute_url': '', 'Title': '…'}
```

Comment:
You are right six.text_type does not make sense here.
But in order to have the title type coherent with the previous implementation I would replace:
```
empty = {'absolute_url': '', 'Title': '…'}
```
with:
```
empty = {'absolute_url': '', 'Title': u'…'}
```